prow/config.yaml

```diff
@@ -467,19 +435,16 @@ presubmits:
         env:
         - name: DOCKER_IN_DOCKER_ENABLED
           value: true
-        # docker-in-docker needs privileged mode
-        securityContext:
-          privileged: true
         volumeMounts:
         - name: docker-graph
           mountPath: /docker-graph
-        ports:
-        - containerPort: 9999
-          hostPort: 9999
+        # docker-in-docker needs privileged mode
```

Comment:
this looks like something we missed earlier but fixing it here since DOCKER_IN_DOCKER_ENABLED is true

Reply:
git diff is being weird here, this was on line 470 previously. what I did here is swap out the hostPath for emptyDir

Reply:
aha. thanks